translations/en.json

```diff
@@ -2247,5 +2247,8 @@
     "until": "until",
     "chat-unmuted-successfully": "Chat unmuted successfully! ",
     "channel-unmuted-successfully": "Channel unmuted successfully! ",
-    "photo-saved": "Photo saved to your device"
+    "photo-saved": "Photo saved to your device",
+    "retake": "Retake",
+    "use-photo": "Use Photo",
+    "PHOTO": "PHOTO"
```

Comment:
that might break other translations then. I think it's best to have the translations do the work

Reply:
Out of context of this PR, but I think we can update `quo/text` component to handle that. We can have 3 variations of a word as `photo`, `Photo`, and `PHOTO`. Instead of having 3 translations of the same word, maybe we can pass a prop to `quo/text` as `first-cap?` and `all-caps?`

Reply:
This is assuming too much about the translations, messages can be very different for different languages so we want to leave that responsibility to the translation file